tests/trailing_commas/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`trailing_commas.php 1`] = `
+<?php
+
+$foo = ['foo', 'bar'];
+$foo = ['very-very-very-very-very-very-very-very-very-long-item-1', 'very-very-very-very-very-very-very-very-very-long-item-2'];
+$foo = ['very-very-very-very-very-very-very-very-very-long-item-1' => 'foo', 'very-very-very-very-very-very-very-very-very-long-item-2' => 'bar'];
+$foo = array('foo', 'bar');
+$foo = array('very-very-very-very-very-very-very-very-very-long-item-1', 'very-very-very-very-very-very-very-very-very-long-item-2');
+$foo = array('very-very-very-very-very-very-very-very-very-long-item-1' => 'foo', 'very-very-very-very-very-very-very-very-very-long-item-2' => 'bar');
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+$foo = ['foo', 'bar'];
+$foo = [
+  'very-very-very-very-very-very-very-very-very-long-item-1',
+  'very-very-very-very-very-very-very-very-very-long-item-2'
+];
+$foo = [
+  'very-very-very-very-very-very-very-very-very-long-item-1' => 'foo',
+  'very-very-very-very-very-very-very-very-very-long-item-2' => 'bar'
+];
+$foo = array('foo', 'bar');
+$foo = array(
+  'very-very-very-very-very-very-very-very-very-long-item-1',
+  'very-very-very-very-very-very-very-very-very-long-item-2'
+);
+$foo = array(
+  'very-very-very-very-very-very-very-very-very-long-item-1' => 'foo',
+  'very-very-very-very-very-very-very-very-very-long-item-2' => 'bar'
+);
+
+`;
+
+exports[`trailing_commas.php 2`] = `
+<?php
+
+$foo = ['foo', 'bar'];
+$foo = ['very-very-very-very-very-very-very-very-very-long-item-1', 'very-very-very-very-very-very-very-very-very-long-item-2'];
+$foo = ['very-very-very-very-very-very-very-very-very-long-item-1' => 'foo', 'very-very-very-very-very-very-very-very-very-long-item-2' => 'bar'];
+$foo = array('foo', 'bar');
+$foo = array('very-very-very-very-very-very-very-very-very-long-item-1', 'very-very-very-very-very-very-very-very-very-long-item-2');
+$foo = array('very-very-very-very-very-very-very-very-very-long-item-1' => 'foo', 'very-very-very-very-very-very-very-very-very-long-item-2' => 'bar');
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+$foo = ['foo', 'bar',];
+$foo = [
+  'very-very-very-very-very-very-very-very-very-long-item-1',
+  'very-very-very-very-very-very-very-very-very-long-item-2',
+];
+$foo = [
+  'very-very-very-very-very-very-very-very-very-long-item-1' => 'foo',
+  'very-very-very-very-very-very-very-very-very-long-item-2' => 'bar',
+];
+$foo = array('foo', 'bar',);
```

Comment:
In https://github.com/prettier/plugin-php/issues/31 we said that if the array fits on one line, we never want trailing commas.

Reply:
@czosel oh yes, stupid error :smile: